mmv1/products/dlp/JobTrigger.yaml

```diff
@@ -912,14 +914,6 @@ properties:
           properties:
             - !ruby/object:Api::Type::NestedObject
               name: 'saveFindings'
-              exactly_one_of:
-                - save_findings
-                - pub_sub
-                - publish_findings_to_cloud_data_catalog
-                - publish_summary_to_cscc
-                - job_notification_emails
-                - deidentify
-                - publish_to_stackdriver
```

Comment:
Do you think it makes sense if we comment this out instead of removing it entirely? We should have ExactlyOneOf restriction on these fields. However, it's not generated correctly due to the feature is blocked by SDK-level issue: https://github.com/hashicorp/terraform-plugin-sdk/issues/470. They should be available once the SDK issue is resolved.

We can add comment: `TODO: uncomment here once they are supported(github.com/hashicorp/terraform-plugin-sdk/issues/470)`

Thanks!

Reply:
Yeah, commenting these `exactly_one_of` blocks makes sense. I've added comments for the same.